tensorizer/serialization.py

```diff
@@ -193,7 +253,7 @@ class _TensorHeaderSerializer:
         "B"  # Hash count (fixed for a particular tensorizer version)
     )
     hash_header_offset: int
-    hash_count: ClassVar[int] = 2
+    hash_count: int
```

Comment:
No, actually, the one that caused issues was [this one in `_bulk_load()`](https://github.com/coreweave/tensorizer/pull/66/files#diff-bed389e293dc56b7ef028c35fe614916c8ae6e31faca6a30f3cc75695447b3f8R2232). Since it always expected 2 hash tasks, if only 1 hash was actually present, it would never reach 0 tasks remaining, so it deadlocked. Worse, if multiple hashes were present, it would continue *before* they had all finished processing, so it's good that the deadlock case from having "too few" hashes came up first.